### ISSUE TITLE
Just a Comment

### ISSUE THREAD

**Issue by tjays7** [brainhack-school2020/stephaniealley_bhs2020_project#1]
Very good idea! I think the preprocessing results may also vary across platforms i.e. which tool (e.g. SPM) was used to perform preprocessing steps.

**Comment by stephaniealley** [comment]
Yes, that is absolutely true. They could also differ with tool version or even with the OS that was used!
